src/main/scala/io/iohk/ethereum/vm/BlockchainConfigForEvm.scala

```diff
@@ -22,9 +24,36 @@ case class BlockchainConfigForEvm(
   aghartaBlockNumber: BigInt,
   phoenixBlockNumber: BigInt,
   chainId: Byte
-)
+){
+  def etcForkForBlockNumber(blockNumber: BigInt): EtcFork = blockNumber match {
+    case _ if blockNumber < atlantisBlockNumber => BeforeAtlantis
+    case _ if blockNumber < aghartaBlockNumber => Atlantis
+    case _ if blockNumber < phoenixBlockNumber => Agharta
+    case _ if blockNumber >= phoenixBlockNumber => Phoenix
+  }
+
+  def ethForkForBlockNumber(blockNumber: BigInt): BlockchainConfigForEvm.EthForks.Value = blockNumber match {
+    case _ if blockNumber < byzantiumBlockNumber => BeforeByzantium
+    case _ if blockNumber < constantinopleBlockNumber => Byzantium
+    case _ if blockNumber >= constantinopleBlockNumber => Constantinople
+
+  }
+}
 
 object BlockchainConfigForEvm {
+
+  object EtcForks extends Enumeration {
+    type EtcFork = Value
+    val BeforeAtlantis, Atlantis, Agharta, Phoenix = Value
+    // TODO add Petersburg and Istanbul
```

Comment:
there is no Petersburg and Istanbul for ETC, or did you mean that add those to config ? If yes then imo it would be better to thiss coment to EthForks

Reply:
You're right. I will move the comment